subsys/nfc/t4t/Kconfig

```diff
@@ -4,6 +4,12 @@
 # SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
 #
 
+config NFC_T4T_NDEF_FILE
+	bool "NFC Type 4 Tag NDEF File"
+	default y if NFC_T4T_NRFXLIB && NFC_NDEF_MSG
```

Comment:
Maybe it should be also default 'y' when `CONFIG_NFC_T4T_HL_PROCEDURE`? Then I guess if one reads the Type 4 tag, he will also want access to the NDEF data through this module.

Reply:
Ok. It seems to be worth adding. Done